src/Microsoft.AspNetCore/WebHost.cs

```diff
@@ -149,12 +149,21 @@ private static IWebHost StartWith(string url, Action<IServiceCollection> configu
         /// <returns>The initialized <see cref="IWebHostBuilder"/>.</returns>
         public static IWebHostBuilder CreateDefaultBuilder(string[] args)
         {
-            var builder = new WebHostBuilder()
-                .UseKestrel((builderContext, options) =>
+            var builder = new WebHostBuilder();
+
+            if (string.IsNullOrEmpty(builder.GetSetting(WebHostDefaults.ContentRootKey)))
+            {
+                builder.UseContentRoot(Directory.GetCurrentDirectory());
```

Comment:
Technically this would be a breaking change so are we okay with taking this @DamianEdwards @davidfowl ? Spoke offline with @glennc and this seems to be okay to take in 2.2.

Reply:
I cleared this with @DamianEdwards offline.